src/components/shared/IssueTicketsButton.tsx

```diff
@@ -75,6 +79,15 @@ export default function IssueTicketsButton({
     )
   }
 
+  const onClose = () => {
+    setModalVisible(false)
+
+    // remove newDeploy=true query parameter
+    history.replace({
```

Comment:
removes the param when closing the modal, so subsequent refreshes of the page don'e trigger the modal